dockerfiles/tensorflow.dockerfile

```diff
@@ -1,14 +1,15 @@
 FROM tensorflow/tensorflow:2.3.0-gpu
 
 RUN apt-get update && \
-    apt-get install -y libsm6 libxrender1 libxext6 libgl1-mesa-glx && \
+    apt-get install -y libsm6 libxrender1 libxext6 libgl1-mesa-glx vim && \
```

Comment:
This docker image should stay as light as possible, and vim shouldn't be specified for all user.
If someone wants to use vim, he should create a new dockerfile for his own purpose